model/src/cfd.rs

```diff
@@ -1860,11 +1863,6 @@ pub fn calculate_long_liquidation_price(
         ContractSymbol::EthUsd => {
             let initial_price = initial_price.to_u64();
 
-            // TODO: this liquidation price is potentially not in-line with the _actual_
```

Comment:
Did you create a ticket for that?

Reply:
Good reminder, I'll do it.

Reply:
Here it is: https://github.com/itchysats/itchysats/issues/2809